app/assets/stylesheets/components/autocomplete.scss

```diff
@@ -57,3 +58,57 @@
     font-size: var(--fs-s);
   }
 }
+
+.c-autocomplete--multi {
+  &__wrapper:focus-within {
+    @extend %form-styling-focus;
+
+    .c-autocomplete--multi__input {
+      background-color: var(--form-bg-focus);
+    }
+  }
+
+  &__input {
+    background-color: var(--form-bg);
+    color: var(--body-color);
+    border: none;
+  }
+
+  .crayons-btn.c-autocomplete--multi__selected {
```

Comment:
I found I needed this level of specificity to be able to override some default crayons styles

Reply:
Out of curiousity, what's the reason for overriding Crayons styles?

Reply:
It's so that we can remove the border radius on the inside corners _and_ change the 'X' colour on hover/focus

Reply:
crayons defines colours in its hover/focus styles already, so I needed to be more specific to override it... I don't think we can do it via crayons styles alone unless we did something really funky like changing the button's variant to `danger` on mouseover, which doesn't feel right.